spec/datetimeformat.html

```diff
@@ -203,33 +209,33 @@ <h1>Internal slots</h1>
           <ul>
-            <li>weekday, year, month, day, hour, minute, second, fractionalSecondDigits</li>
-            <li>weekday, year, month, day, hour, minute, second</li>
-            <li>weekday, year, month, day</li>
-            <li>year, month, day</li>
-            <li>year, month</li>
-            <li>month, day</li>
-            <li>hour, minute, second, fractionalSecondDigits</li>
-            <li>hour, minute, second</li>
-            <li>hour, minute</li>
-            <li>dayPeriod, hour</li>
-            <li>dayPeriod, hour, minute, second</li>
-            <li>dayPeriod, hour, minute</li>
+            <li>[[Weekday]], [[Year]], [[Month]], [[Day]], [[Hour]], [[Minute]], [[Second]], [[FractionalSecondDigits]]</li>
+            <li>[[Weekday]], [[Year]], [[Month]], [[Day]], [[Hour]], [[Minute]], [[Second]]</li>
+            <li>[[Weekday]], [[Year]], [[Month]], [[Day]]</li>
+            <li>[[Year]], [[Month]], [[Day]]</li>
+            <li>[[Year]], [[Month]]</li>
+            <li>[[Month]], [[Day]]</li>
+            <li>[[Hour]], [[Minute]], [[Second]], [[FractionalSecondDigits]]</li>
+            <li>[[Hour]], [[Minute]], [[Second]]</li>
+            <li>[[Hour]], [[Minute]]</li>
+            <li>[[DayPeriod]], [[Hour]]</li>
+            <li>[[DayPeriod]], [[Hour]], [[Minute]], [[Second]]</li>
+            <li>[[DayPeriod]], [[Hour]], [[Minute]]</li>
           </ul>
           Each of the records must also have the following fields:
           <ol>
-            <li>A [[pattern]] field, whose value is a String value that contains for each of the date and time format component fields of the record a substring starting with *"{"*, followed by the name of the field, followed by *"}"*.</li>
-            <li>If the record has an [[hour]] field, it must also have a [[pattern12]] field, whose value is a String value that, in addition to the substrings of the [[pattern]] field, contains at least one of the substrings *"{ampm}"* or *"{dayPeriod}"*.</li>
-            <li>If the record has a [[year]] field, the [[pattern]] and [[pattern12]] values may contain the substrings *"{yearName}"* and *"{relatedYear}"*.</li>
+            <li>A [[Pattern]] field, whose value is a String value that contains for each of the date and time format component fields of the record a substring starting with *"{"*, followed by the name of the field, followed by *"}"*.</li>
+            <li>If the record has an [[Hour]] field, it must also have a [[Pattern12]] field, whose value is a String value that, in addition to the substrings of the [[Pattern]] field, contains at least one of the substrings *"{ampm}"* or *"{dayPeriod}"*.</li>
+            <li>If the record has a [[Year]] field, the [[Pattern]] and [[Pattern12]] values may contain the substrings *"{yearName}"* and *"{relatedYear}"*.</li>
```

Comment:
Unfortunately, capitalizing the field names in [[LocaleData]].[[<_locale_>]].[[Formats]].[[<_calendar_>]] disrupts this and related mechanisms because of the coupling between field name and brace-enclosed field value substrings (e.g., an "{hour}:{minute} {ampm}" pattern is specifically referencing fields named [[hour]], [[minute]], and [[ampm]]). It's probably most expedient to leave alone any field named after a unit/date–time element/etc., unless you want to do a deeper dive on the format patterns and FormatDateTimePattern's transformation of them.

Reply:
blast, I should have caught that!  I'm guessing it's correct to assume that a deeper dive on FormatDateTimePattern should be understood as a very low priority.